lib/modules/removeRedundantAttributes.es6

```diff
@@ -32,12 +32,11 @@ const redundantAttributes = {
     }
 };
 
+const TAG_MATCH_REGEXP = new RegExp('^(' + Object.keys(redundantAttributes).join('|') + ')$');
```

Comment:
Since `redundantAttributes` stays the same, which means `Object.keys(redundantAttributes)` and `new RegExp('^(' + tags.join('|') + ')$')` will remain the same. So there is no need to build `tagMatchRegExp` everytime the function is called.